flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -780,7 +780,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
     }
 
     set inOut(v) {
-        if (v.length < 23) {
+        if (v === null || v.length < 23) {
```

Comment:
What if v is undefined? How about also checking for typeof to catch undefined etc. as well:

```suggestion
        if (v === null || (typeof v === "string" && v.length < 23)) {
```